### PR TITLE
port Common toolchains page to MarkDown

### PR DESCRIPTION
**Pull request by orbsmiv**
- [x] Update the many cross-links

### CODE REVIEW

File: docs/common-toolchains.md [hunk 1]
```diff
@@ -1,293 +1,212 @@
...
+To customize a toolchain component, you should copy the corresponding
+easyconfig file, modify according to your needs, and make sure it is
+included in a location in the robot search path that precedes the
+location of the easyconfig files that are included with EasyBuild (see
+also [Searching for easyconfigs: the robot search path](using-easybuild.md#searching-for-easyconfigs-the-robot-search-path)), before building and
+installation the toolchain.
+
+!!! info "More information about toolchains"
+
+    Please see the [List of known toolchains](version-specific/eb-list-toolchains.md#list-of-known-toolchains) for how to obtain a listing of
```
Comment (orbsmiv) [inline docs/common-toolchains.md:208]: I've added links in this section that point at content within the auto-generated `version-specific` directory. I've taken the placeholders from the current docs' pages but is this the correct thing to do? Obviously these placeholders don't yet exist as we're deliberately not editing any content in this directory.

Comment (orbsmiv) [reply docs/common-toolchains.md]: Resolved by 9ef3db1

**Comment by orbsmiv** [comment]
Need to update internal headers, moving the --commands to the next line, where required

File: docs/common-toolchains.md [hunk 2]
```diff
@@ -1,293 +1,212 @@
...
+
+| `intel` | *date*   | *binutils* | *GCC*  | *Intel compilers* | *Intel MPI* | *Intel MKL* |
+|---------|----------|------------|--------|-------------------|-------------|-------------|
+| `2019a` | Jan '19  | 2.31.1     | 8.2.0  | 2019.1.144        | 2018.4.274  | 2019.1.144  |
+| `2019b` | Sept '19 | 2.32       | 8.3.0  | 2019.5.281        | 2018.5.288  | 2019.5.281  |
+| `2020a` | May'20   | 2.34       | 9.3.0  | 2020.1.217        | 2019.7.217  | 2020.1.217  |
+| `2020b` | Nov'20   | 2.35       | 10.2.0 | 2020.4.304        | 2019.9.304  | 2020.4.304  |
+| `2021a` | May'21   | 2.36.1     | 10.3.0 | 2021.2.0          | 2021.2.0    | 2021.2.0    |
+| `2021b` | Oct'21   | 2.37       | 11.2.0 | 2021.4.0          | 2021.4.0    | 2021.4.0    |
+| `2022a` | Jun'22   | 2.38       | 11.3.0 | 2022.1.0          | 2021.6.0    | 2022.1.0    |
```
Comment (branfosj) [inline docs/common-toolchains.md:161]:
```suggestion
| `2020a` | May '20   | 2.34       | 9.3.0  | 2020.1.217        | 2019.7.217  | 2020.1.217  |
| `2020b` | Nov '20   | 2.35       | 10.2.0 | 2020.4.304        | 2019.9.304  | 2020.4.304  |
| `2021a` | May '21   | 2.36.1     | 10.3.0 | 2021.2.0          | 2021.2.0    | 2021.2.0    |
| `2021b` | Oct '21   | 2.37       | 11.2.0 | 2021.4.0          | 2021.4.0    | 2021.4.0    |
| `2022a` | Jun '22   | 2.38       | 11.3.0 | 2022.1.0          | 2021.6.0    | 2022.1.0    |
```

File: docs/common-toolchains.md [hunk 3]
```diff
@@ -1,293 +1,212 @@
...
+also [Searching for easyconfigs: the robot search path][robot_search_path]), before building and
+installation the toolchain.
+
+!!! info "More information about toolchains"
+
+    Please see the [List of known toolchains](version-specific/eb-list-toolchains.md) for how to obtain a listing of
+    the currently known toolchains.
+
+    For a detailed listing of the compiler options available with each
+    toolchain, please see [Available toolchain options (by toolchain)](version-specific/toolchain-opts.md).
```
Comment (branfosj) [inline docs/common-toolchains.md:212]:
```suggestion
## More information about toolchains

Please see the [List of known toolchains](version-specific/eb-list-toolchains.md) for how to obtain a listing of
the currently known toolchains.

For a detailed listing of the compiler options available with each
toolchain, please see [Available toolchain options (by toolchain)](version-specific/toolchain-opts.md).
```

**Review by branfosj — CHANGES_REQUESTED** [review]

**Review by branfosj — APPROVED** [review]

**Pull request merged**